docs/advanced/admin-commands.md

```diff
@@ -1,58 +1,245 @@
+
+Rights statements files must be tab-separated files where each line contains a
+single copyright statement's data in the format of URI followed by a [TAB]
+followed by a label.  e.g.:
+
+```
+http://www.europeana.eu/rights/rr-f/	Rights Reserved - Free Access
+http://creativecommons.org/licenses/by/4.0/	Attribution 4.0 International (CC BY 4.0)
+http://creativecommons.org/licenses/by-nc/4.0/	Attribution-NonCommercial 4.0 International (CC BY-NC 4.0)
+http://creativecommons.org/licenses/by-nc-nd/4.0/	Attribution-NonCommercial-NoDerivatives 4.0 International (CC BY-NC-ND 4.0)
 ```
 
-- batches
-- commit_index
-- delete_cache
-- diff_batches
-- dump_ocr
-- ethnicities_with_issues
-- index
-- index_pages
-- index_titles
-- link_places
-- [load_batch](#load_batch)
-- load_batches
-- load_copyright
-- load_copyright_map
-- [load_titles](#load_titles)
-- make_countries_fixture
-- process_coordinates
-- purge_batch
-- purge_django_cache
-- purge_etitles
-- reconcile
-- setup_index
-- update_has_issues
-- update_sitemap
-- zap_index
-
-## load_batch
-
-Please refer to the
-[Load and Purge Batches documentation](/docs/manage-data/batches-load-purge.md#load-batch)
-for detailed information about this command.
-
-## load_titles
-
-Documentation pending.
-
-## purge_batch
-
-Please refer to the
-[Load and Purge Batches documentation](/docs/manage-data/batches-load-purge.md#purge-batch)
-for detailed information about this command.
\ No newline at end of file
+Please note that loading the same file multiple times will result in duplicated
+data, and manual SQL may be needed to clean dupes from `core_copyright`.
+
+## `load_copyright_map`
+
+Defines rules for which titles should use a given rights statement for certain
+date ranges.  Rights *must* first be loaded via
+[`load_copyright`](#load_copyright).
+
+Rules are composed of four-field tab-separated-values files, where each line
+indicates a single rule.  The fields, in order, are LCCN, start date, end date,
+and rights URI.  Start and end dates must be formatted as `YYYY-MM-DD`, e.g.,
+`2001-09-08` means September 8th, 2001.
+
+An example rules file might look like this:
+
+```
+sn83045462	1922-01-01	1999-12-31	http://creativecommons.org/licenses/by-nc/4.0/
+sn83045462	2000-01-01	2999-12-31	http://www.europeana.eu/rights/rr-f/
+```
+
+Please note that loading the same file multiple times will result in duplicated
+data, and manual SQL may be needed to clean dupes from `core_lccndatecopyright`.
+
+## `load_titles`
+
+Loads titles into Open ONI from a single file or a path containing multiple
+files.  All files must contain valid MARC XML.
+
+A typical example might look like this:
+
+```bash
+./manage.py load_titles /var/local/marcxml
+```
+
+You can aggregate custom / overridden MARC XML in a single place and run a
+command like this as often as necessary; titles which already exist will not be
+reimported.  (If you *need* to overwrite a title, there is unfotunately no way
```

Comment:
typo in "unfotunately"

Reply:
Wow, that's really unfotunate.  I'll fix it.